arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -261,5 +263,41 @@ public async Task GetCollectionPropertiesAsync_ShouldSucceed()
             Assert.Equal(3, response.Status);
             Assert.Equal(2, response.Type);
         }
+
+        [Fact]
+        public async Task PutCollectionPropertyAsync_ShouldSucceed()
+        {
+            var putCollection = await _adb.Collection.PostCollectionAsync(new PostCollectionBody
+            {
+                 Name = nameof(PutCollectionPropertyAsync_ShouldSucceed)
+            });
+            var beforeResponse = await _collectionApi.GetCollectionPropertiesAsync(putCollection.Name);
+
+            var body = new PutCollectionPropertyBody
+            {
+                WaitForSync = !beforeResponse.WaitForSync
+            };
+            var response = await _collectionApi.PutCollectionPropertyAsync(putCollection.Name, body);
+
+            Assert.Equal(HttpStatusCode.OK, response.Code);
+            Assert.NotEqual(beforeResponse.WaitForSync, response.WaitForSync);
+        }
+
+        [Fact]
+        public async Task PutCollectionPropertyAsync_ShouldThrow_WhenNotFound()
```

Comment:
Minor thing: maybe say what should be "not found"? Example: `PutCollectionPropertyAsync_ShouldThrow_WhenCollectionNotFound` or `PutCollectionPropertyAsync_ShouldThrow_WhenCollectionDoesNotExist`